docs/community-operators-troubleshooting.md

```diff
@@ -0,0 +1,219 @@
+* [dockerfile-creation](#dockerfile-creation)
+* [build-bundle](#build-bundle)
+* [make-bundle-repo-public](#make-bundle-repo-public)
+* [get-supported-versions](#get-supported-versions)
+* [add-bundle-to-index](#add-bundle-to-index)
+* [make-index-repo-public](#make-index-repo-public)
+* [get-ci-results-attempt](#get-ci-results-attempt)
+* [preflight-trigger](#preflight-trigger)
+* [evaluate-preflight-result](#evaluate-preflight-result)
+* [get-ci-results](#get-ci-results)
+* [link-pull-request-with-open-status](#link-pull-request-with-open-status)
+* [merge-pr](#merge-pr)
+* [link-pull-request-with-merged-status](#link-pull-request-with-merged-status)
+
+### [Operator Released Pipeline](#released)
+* [set-github-started-label](#set-github-started-label)
+* [set-github-status-pending](#set-github-status-pending)
+* [set-env](#set-env)
+* [clone-repository-base](#clone-repository-base)
+* [clone-repository](#clone-repository)
+* [detect-changes](#detect-changes)
+* [read-config](#read-config)
+* [resolve-pr-type](#resolve-pr-type)
+* [content-hash](#content-hash)
+* [certification-project-check](#certification-project-check)
+* [get-organization](#get-organization)
+* [get-pyxis-certification-data](#get-pyxis-certification-data)
+* [copy-bundle-image-to-released-registry](#copy-bundle-image-to-released-registry)
+* [get-supported-versions](#get-supported-versions)
+* [acquire-lease](#acquire-lease)
+* [add-bundle-to-index](#add-bundle-to-index)
+* [decide-index-paths](#decide-index-paths)
+* [get-manifest-digests](#get-manifest-digests)
+* [request-signature](#request-signature)
+* [upload-signature](#upload-signature)
+* [publish-to-index](#publish-to-index)
+
+
+# <a id="hosted"></a>Operator Hosted Pipeline
+
+## <a id="get-pr-number"></a>get-pr-number
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="acquire-lease"></a>acquire-lease
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="set-github-started-label"></a>set-github-started-label
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="set-github-status-pending"></a>set-github-status-pending
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="set-env"></a>set-env
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="clone-repository-base"></a>clone-repository-base
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="clone-repository"></a>clone-repository
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="detect-changes"></a>detect-changes
```

Comment:
Please describe a possible reasons why this task fails (update of existing bundles, more than one bundle submitted, ...). Also, suggest reading logs to understand what the failure causes are.